src/test/java/teammates/test/pageobjects/InstructorFeedbacksPage.java

```diff
@@ -469,6 +469,20 @@ public void verifyUnpublishLinkHidden(String courseId, String sessionName) {
         }
     }
 
+    public boolean verifyPublishLinkDisabled(String courseId, String sessionName) {
```

Comment:
You may want to test if the caret button is disabled instead (possibly add another class for the caret button as well) and add some documentation about why is the caret button is tested instead.